test/integration/infrastructure/adapters/GitCasWarpStateCacheAdapter.retention.integration.test.ts

```
@@ -0,0 +1,142 @@
+  });
+}
+
+async function prunableOids(repoPath: string): Promise<Set<string>> {
+  const { stdout: output } = await execFileAsync(
+    'git',
+    ['-C', repoPath, 'prune', '-n', '--expire=now'],
+  );
+  return new Set(
+    output
+      .split('\n')
+      .map((line) => line.trim().split(/\s+/)[0])
```

ESLint

[error] Move this regular expression to module scope to avoid re-compilation on every call.

(e18e/prefer-static-regex)